qutip/solver/propagator.py

```diff
@@ -61,12 +61,15 @@ def propagator(H, t, c_ops=(), args=None, options=None, **kwargs):
     if not isinstance(H, (Qobj, QobjEvo)):
         H = QobjEvo(H, args=args, **kwargs)
 
-    if H.issuper or c_ops:
-        out = mesolve(H, qeye(H.dims), tlist, c_ops=c_ops,
```

Comment:
Did this code do the wrong thing when `c_ops` were present?

Reply:
The code was wrong when `H` was a super operator. It that case it should have been `qeye(H.dims[0])`.